app/src/main/java/org/jellyfin/mobile/player/PlayerFragment.kt

```diff
@@ -443,14 +445,16 @@ class PlayerFragment : Fragment() {
 
             override fun onScale(detector: ScaleGestureDetector): Boolean {
                 val scaleFactor = detector.scaleFactor
-                if (abs(scaleFactor - 1f) > 0.01f) {
+                if (abs(scaleFactor - Constants.ZOOM_SCALE_BASE) > Constants.ZOOM_SCALE_THRESHOLD) {
                     isZoomEnabled = scaleFactor > 1
                     updateZoomMode(isZoomEnabled)
                 }
                 return true
             }
 
-            override fun onScaleEnd(detector: ScaleGestureDetector) {}
+            @Suppress("EmptyFunctionBlock")
+            override fun onScaleEnd(detector: ScaleGestureDetector) {
+            }
```

Comment:
You might be able to do:
```suggestion
            override fun onScaleEnd(detector: ScaleGestureDetector) = Unit
```
or
```suggestion
            override fun onScaleEnd(detector: ScaleGestureDetector) = Nothing
```